pkg/procmanager/procmanager.go

```diff
@@ -31,6 +31,30 @@ func NewProcManager(dryRun bool) *ProcManager {
 	}
 }
 
+// RunRepeatWithRetries performs a Run multiple times with retries.
+// If an error occurs during the repetition, the last error will be returned.
+func (pm *ProcManager) RunRepeatWithRetries(
+	ctx context.Context,
+	args []string,
+	stdIn io.Reader,
+	stdOut io.Writer,
+	stdErr io.Writer,
+	cmdDir string,
+	retries int,
+	retryDelay int,
+	containerName string,
+	repeat int,
+	ignoreErrors bool) error {
+	var err error
+	for i := 0; i <= repeat; i++ {
+		innerErr := pm.RunWithRetries(ctx, args, stdIn, stdOut, stdErr, cmdDir, retries, retryDelay, containerName)
+		if innerErr != nil {
+			err = innerErr
```

Comment:
wrap the err from each iteration and return the wrapped one

Reply:
Switched it to use our internal `errors[]` which has a `String()` method that does `string.Join(",")` on the aggregated list.